MozziHeadersOnly.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Mozzi.h
+ *
+ * Copyright 2023, Thomas Combriat and the Mozzi team
+ *
+ * This file is part of Mozzi.
+ *
+ * Mozzi is licensed under a Creative Commons
+ * Attribution-NonCommercial-ShareAlike 4.0 International License.
+ *
+ */
+
+/** @ingroup core 
+ * @file MozziHeadersOnly.h
+ *
+ * This file provides declarations of the \ref core Mozzi functions, but no implementation. Use this only, if you have more than one
```

Comment:
So the idea is that Mozzi.h should always be included in the main sketch and MozziHeaderOnly.h in every user cpp file?

Reply:
At least in every .cpp file that needs Mozzi delcarations, yes. I added a minimal example to that effect as Basics->Skeleton_Multi